evap/evaluation/templates/notebook.html

```diff
@@ -0,0 +1,30 @@
+<div class="notebook">
+   <div class="collapse position-fixed collapse-horizontal notebook-animation mt-0 d-print-none" tabindex="-1" id="notebook" aria-expanded="false">
+        <div class="card">
+            <div class="offcanvas-header notebook-button pt-3 ps-3 pe-3">
+                <h5 class="card-title m-0">
+                    {% trans 'Notebook' %}
+                </h5>
+                <button type="button" class="btn-close text-reset" data-bs-toggle="collapse" data-bs-target="#notebook" aria-controls="collapseNotebook" id="notebookButtonClose"></button>
```

Comment:
`document.getElementById("collapseNotebook")` returns `null` for me, both when the notebook is open and when it is closed. What is this intended to do?